src/main/resources/templates/eac/officerInformation.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+    <title>Officer Information</title>
+</head>
+<div id="temp-home-main-content"  layout:fragment="content">
+
+    <form th:action="@{''}" method="post" class="form" id="home">
```

Comment:
can you change the id from home to something more specific?

Reply:
Will do